provisioning_client/samples/prov_dev_client_sample/prov_dev_client_sample.c

```diff
@@ -103,6 +103,8 @@ int main()
     (void)prov_dev_security_init(hsm_type);
 
     // Set the symmetric key if using they auth type
+    //If using DPS with an enrollment group, this must the the derived key from the DPS Master key
+    //https://docs.microsoft.com/en-us/azure/iot-dps/how-to-legacy-device-symm-key#derive-a-device-key
```

Comment:
We could point to the updated documentation here instead: https://docs.microsoft.com/en-us/azure/iot-dps/concepts-symmetric-key-attestation?tabs=azure-cli#group-enrollments